### DISCUSSION TITLE
[Python | NumberRange] When to support number range in python?

### DISCUSSION THREAD

**Discussion by lexmen318** [microsoft/Recognizers-Text#2056] [Q&A]
I really need a python package to parse number range info from text.

Thx a lot!

**Comment by tellarin** [comment]
Hi @lexmen318,
the Python packages are still in alpha. Which language are you targeting? English first? 
Would you be able to help contributing with the implementation?

**Comment by lexmen318** [comment]
Thank you very very much for your quick response!
My first priority on language is Chinese, and second one is English.

**Comment by tellarin** [comment]
Would you be able to help contributing with the implementation?
